### DISCUSSION TITLE
Issue with dragging Region when also using the Envelope plugin

### DISCUSSION THREAD

**Discussion by alveslionel03** [katspaugh/wavesurfer.js#3665] [Q&A]
Hello everyone,
I'm trying to integrate wavesurfer into one of my web applications. I'm using both the Envelope and Region plugins, specifically the markers.
 It seems that the Regions cannot be dragged when the Envelope plugin is also used. 
 I assume this is the same issue as the one seen in the multimix example, where the Regions cannot be dragged:
 https://wavesurfer.xyz/examples/?multitrack.js
 
 Have I missed any information? Is this an intended behavior? How can I make the Regions draggable?

Thank you!

**Comment by matthewboman** [comment]
As a workaround, I changed the `z-index` of the regions container to 6 (it's 4 for the Polyline `<svg>` that Envelope uses). The Envelope isn't adjustable when the Region is over it, but you can move the Region to adjust the Envelope.

**Comment by katspaugh** [comment]
That sounds like a bug, please feel free to push this workaround as a PR.
